flipperdevices/Flipper-Android-App
# Remotecontrols/add errors

**Background**

On remote controls screen we don't have information about flipper-related errors

**Changes**

- Add decorated InfraredBackendApi to handle flipper-related errors

**Test plan**

- Open remote controls with disconnected flipper/disabled internet/wrong firmware
- See full-screen errors
